enn_ppo/enn_ppo/train.py

```diff
@@ -336,6 +342,14 @@ def run(
                 self.actions.extend(action)
                 self.logprobs.extend(logprob)
 
+            if capture_videos:
+                self.rendered = np.concatenate(
```

Comment:
This looks like it will take O(n^2) time

Reply:
I didnt think concatenate was O(n)... I can change this to do a single np.concatenate at the end?

Reply:
I don't actually know how concatenate is implemented, but my first assumption would have been it needs to allocate a new array and copy the two source arrays. Single concatenate at the end should work.